### PR TITLE
Full command settings

### PR DESCRIPTION
**Pull request by BroneCk**
New PR as requested :)

### CODE REVIEW

**Comment by Swoffa** [comment]
Thanks!

File: Preferences.sublime-settings [hunk 1]
```diff
@@ -8,5 +8,11 @@
...
+	"kickass_debug_command_x64": "\"${kickass_debug_path}\" -logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${kickass_output_path}/${build_file_base_name}_MonCommands.mon\" ${kickass_debug_args} \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_run_command_x64": "\"${kickass_run_path}\" -logfile \"${kickass_output_path}/${build_file_base_name}_ViceLog.txt\" -moncommands \"${kickass_output_path}/${build_file_base_name}.vs\" ${kickass_run_args} \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_run_command_c64debugger": "\"${kickass_run_path}\" -autojmp -layout 1 -symbols \"${kickass_output_path}/${build_file_base_name}.vs\" -wait 2500 -prg \"${kickass_output_path}/${build_file_base_name}.prg\"",
+	"kickass_compile_parameters": "\"${build_file_base_name}.${file_extension}\" -log \"${kickass_output_path}/${build_file_base_name}_BuildLog.txt\" -o \"${kickass_output_path}/${build_file_base_name}.prg\" -vicesymbols -showmem -symbolfiledir \"${kickass_output_path}\" ${kickass_args}",
```
Comment (Swoffa) [inline Preferences.sublime-settings]: Rename to `kickass_compile_args` for consistency

File: Preferences.sublime-settings [hunk 2]
```diff
@@ -8,5 +8,11 @@
...
+	"kickass_compile_debug_additional_parameters": "-afo :afo=true :usebin=true",
```
Comment (Swoffa) [inline Preferences.sublime-settings]: Rename to `kickass_compile_debug_additional_args` for consistency

Comment (BroneCk) [reply Preferences.sublime-settings]: done

File: Preferences.sublime-settings (hunk 1 above)
Comment (BroneCk) [reply Preferences.sublime-settings]: done

**Comment by Swoffa** [comment]
Merged.
Change to breakpoints file instead of dbg-file for `kickass_debug_command_c64debugger`.
As i understand:
- .dbg-file is automatically loaded without args in version  0.64.56 of c64debugger
- dbg-files overrides both `-breakpoints` and `-symbols`

**Comment by BacchusFLT** [comment]
Simon,

What's the idea between fixes and features implemented in the main branch
and making versions available in the Sublime package repo?

/Pontus Berg
 Bergatrollet AB
 CEO & Owner
 Tel/SMS: +46 735 082860
 Skype: BacchusBerg
 Hangout: pontus.bergatrollet@gmail.com
 www.bergatrollet.se


Den ons 21 nov. 2018 kl 08:41 skrev Simon Oskarsson <
notifications@github.com>:

**Comment by Swoffa** [comment]
> Simon, What's the idea between fixes and features implemented in the main branch and making versions available in the Sublime package repo? /Pontus Berg

@BacchusFLT 
The idea is to make changes available fast in package control as well...which i failed to do for a while now. Sorry for that. Just need to fix release notes to be understandable. Thanks for encouraging.

**Comment by BacchusFLT** [comment]
Between stable versions I'd say you don't need to rush but as long that the
queue is merely feature requests and not really bugs, I'd say that's when
to push a new version.

/Pontus Berg
 Bergatrollet AB
 CEO & Owner
 Tel/SMS: +46 735 082860
 Skype: BacchusBerg
 Hangout: pontus.bergatrollet@gmail.com
 www.bergatrollet.se


Den ons 21 nov. 2018 kl 11:39 skrev Simon Oskarsson <
notifications@github.com>:

**Comment by Swoffa** [comment]
> Between stable versions I'd say you don't need to rush but as long that the queue is merely feature requests and not really bugs, I'd say that's when to push a new version. /Pontus Berg 

@BacchusFLT 
Version 1.3.0 out now thru package control. \o/

**Pull request merged**